src/formpack/reporting/export.py

```diff
@@ -471,17 +436,12 @@ def _get_value_from_entry(
             row.update(_empty_row)
 
             attachments = entry.get('_attachments') or attachments
-            supplemental_details = (
-                entry.get('_supplementalDetails') or supplemental_details
-            )
 
             for field in _fields:
                 # TODO: pass a context to fields so they can all format ?
                 if field.can_format:
                     # get submission value for this field
-                    val = _get_value_from_entry(
-                        entry, field, supplemental_details
-                    )
+                    val = field.get_value_from_entry(entry)
```

Comment:
This is a new method on the `FormField` class that can be overridden as needed by child classes